docs/ops/config.rst

```diff
@@ -255,6 +255,16 @@ in other Django projects.
    If :envvar:`DJANGO_USE_OIDC` is set to ``True``, this settings must be set to
    the URL that a user can visit to logout. It may be a relative URL.
 
+.. envvar:: DJANGO_PEER_APPROVAL_ENFORCED
```

Comment:
I wonder if this should be named something else. My expectation with this is it disables peer approval altogether when set to `True`.